### ISSUE TITLE
Consider using drop shadows for images for visibility

### ISSUE THREAD

**Issue by wwarriner** [uabrc/uabrc.github.io#110]
CSS Hacks: https://github.com/squidfunk/mkdocs-material/issues/1635

Drop Shadow CSS function documentation: https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/drop-shadow()

![image](https://user-images.githubusercontent.com/6930772/156661004-9d16dd0f-b2bf-40eb-90d1-717b947e07b1.png)

The change would be made in [extra.css](https://github.com/uabrc/uabrc.github.io/blob/fc29dba/docs/stylesheets/extra.css)

**Comment by jgordini** [comment]
Drop shadows can be confusing to the user as they typically are used on clickable buttons.

Referencing this article. https://graphicdesign.stackexchange.com/a/137967

**Comment by wwarriner** [comment]
My thought process on this is that the images don't stand out from the page background in many cases, nor from the text, so they can be hard to parse out from the page. Have any thoughts on alternatives to a drop shadow for that use case?

**Comment by jgordini** [comment]
Totally agree does need some differentiation. 
possible alternative to drop shadow is adding borders. 

border-radius: 0.25rem;
border: 0.1rem solid #e5e7eb

example is attached. 

![Screen Shot 2022-03-08 at 1 55 04 PM](https://user-images.githubusercontent.com/92876980/157314788-0b902096-8e02-4311-a62b-3e966a44ad33.png)

**Comment by wwarriner** [comment]
Oh that looks nice. Just in case you haven't seen it, the file to make the change for mkdocs is [docs/stylesheets/extra.css](https://github.com/uabrc/uabrc.github.io/blob/main/docs/stylesheets/extra.css) if you wanted to make a pull request or play around with it on your local repo.

**Comment by jgordini** [comment]
Possible solution is using everything at once. Allows for less drop shadow and still diferentiates. :)
Will play around. Example below. 

    border-radius: 0.25rem;
    border: 0.1rem solid #e5e7eb;
    box-shadow: 0 4px 6px -1px rgb(0 0 0 / 10%), 0 2px 4px -2px rgb(0 0 0 / 10%);

![Screen Shot 2022-03-08 at 2 05 47 PM](https://user-images.githubusercontent.com/92876980/157316412-5e302707-81e2-493f-a59d-ec78355acb5e.png)

**Comment by jgordini** [comment]
Looked the best with just drop shadow. Will try and work up that option.

**Comment by jgordini** [comment]
#125 added drop shadow

**Comment by jgordini** [comment]
you can see the changes I have made here. https://jgordini.github.io/uabrc.github.io/uab_cloud/network_setup_basic/

**Comment by jgordini** [comment]
#125 referenced in this pull request.

**Comment by wwarriner** [comment]
Fixed by #209
